Meeqan/buys
# #162832764 Implement email verifications

**What does this PR do?**

This PR implements the User email verification endpoints.

**Description of Task to be completed?**
As a developer, I should be able to send out a clickable verify token to the registered email by making a POST request with the valid details to /user/register.

**How should this be manually tested?**
- Pull the branch `git fetch origin  ft-email-verification-162832764`
- Check into it` git checkout  ft-email-verification-162832764`
- Activate the pipenv
- Run the app: python manage.py runserver.
- On postman make a POST request to `{{base-url}}/api/v1/user/register`.
- Check your email for a verification link.

body - Register
```
{
    "email": "example@gmail.com",
    "password": "Password@123",
    "first_name": "Test",
    "last_name": "User"
}
```

**Any background context you want to provide?**
N/A

**What are the relevant pivotal tracker stories?**
[#162832764](https://www.pivotaltracker.com/n/projects/2170023/stories/162832764)
